install/kubernetes/helm/istio/values.yaml

```diff
@@ -40,6 +40,21 @@ global:
     # If set, newly injected sidecars will have core dumps enabled.
     enableCoreDump: false
 
+    # Default port for Pilot agent health checks. A value of 0 will disable health checking.
+    statusPort: 15003
+
+    # Default HTTP path for readiness probes on the statusPort.
+    readinessPath: /healthz/ready
+
+    # The initial delay for readiness probes in seconds.
+    readinessInitialDelaySeconds: 2
+
+    # The period between readiness probes.
+    readinessPeriodSeconds: 5
+
```

Comment:
no liveness?

Reply:
Be careful not to fall into the trap of setting the same thing for liveness as you do readiness.   Liveness should be for an unrecoverable state, from which you could infer that the app if it was in an unrecoverable state should exit with a non-zero status code anyway (and thus, get restarted).

Reply:
@rshriram liveness is not being covered by this PR.  @mandarjog will be picking up the liveness work after this PR goes in.